devtools/create_project/codeblocks.cpp

```diff
@@ -58,8 +58,11 @@ StringList getFeatureLibraries(const BuildSetup &setup) {
 	for (FeatureList::const_iterator i = setup.features.begin(); i != setup.features.end(); ++i) {
 		if (i->enable && i->library) {
 			std::string libname;
-			if (!std::strcmp(i->name, "libz") || !std::strcmp(i->name, "libcurl")) {
+			if (!std::strcmp(i->name, "libcurl")) {
 				libname = i->name;
+			} else if (!std::strcmp(i->name, "zlib")) {
+				libname = "libz";
+				libraries.push_back("libvorbisfile");
```

Comment:
looks like libvorbisfile shouldn't be here

Reply:
Whopsie.